apps/server/src/modules/school/uc/school.uc.ts

```diff
@@ -43,6 +48,7 @@ export class SchoolUc {
 
 		if (!existingUserLoginMigration?.closedAt && updatedUserLoginMigration.closedAt) {
 			await this.schoolMigrationService.markUnmigratedUsersAsOutdated(schoolId);
+			await this.userLoginMigrationRollbackService.rollbackIfNecessary(schoolId);
```

Comment:
we should not mark unmigrated users as outdated when we roll back. the "rollback" is more about preventing any unwanted action on closing the migration and deleting the UserLoginMigration, when the school was not migrated. It is actually less of a "rollback" then a "cancel" ,since nothing has been done yet (or should be done).